vhdbuilder/packer/init-variables.sh

```diff
@@ -334,18 +338,18 @@ if [ "$OS_TYPE" == "Windows" ]; then
 		WINDOWS_IMAGE_VERSION=""
 
 		# Need to use a sig image to create the build VM
-		if [[ "$MODE" == "sigMode" ]]; then
-			# Reuse IMPORTED_IMAGE_NAME so the shared code in cleanup.sh can delete the temporary resource
-			IMPORTED_IMAGE_NAME=$imported_windows_image_name
-			echo "Creating new image for imported vhd ${WINDOWS_IMAGE_URL}"
+		if [[ "$MODE" == "windowsVhdMode" ]]; then
+			# a. create a new managed image $IMPORTED_IMAGE_NAME from os disk source $IMPORTED_IMAGE_URL
```

Comment:
Suggest not using a. b. c. d. in multiple comments and it need additional work to update them if another step is added in future.